source/Random.cpp

```diff
@@ -104,12 +109,53 @@ uint32_t Random::Binomial(uint32_t t, double p)
 
 
 
-// Get a normally distributed number (mean = 0, sigma= 1).
-double Random::Normal()
+// Get a normally distributed number (mean = 0, sigma= 1) using std::normal_distribution.
+double Random::StdNormal()
 {
 	normal_distribution<double> normal;
 #ifndef __linux__
 	lock_guard<mutex> lock(workaroundMutex);
 #endif
 	return normal(gen);
 }
+
+
+
+// Get a normally distributed number (mean = 0, sigma= 1) using std::normal_distribution.
+double Random::StdNormal(double mean, double sigma)
+{
+	normal_distribution<double> normal{mean, sigma};
+#ifndef __linux__
+	lock_guard<mutex> lock(workaroundMutex);
+#endif
+	return normal(gen);
```

Comment:
Reading this again, I'm realizing I can't really agree with this.  If the previous Normal function is doing more as a rule (in the shipped code) then it needs to be tested accordingly.  If it can be modified to pre-generate the distributions as a rule, then it can be tested with that in mind.

Reply:
Likewise, however, if my implementation of a Box-Muller normal is not doing enough, a new comparison would be fair.

Reply:
> (You can run benchmarks automatically via the unit tests, btw. There are existing benchmarks for the members of Random.)

I couldn't get them to run after trying a couple different things:
-  Successfully linking the includes in a different directory.
- Attempting to compile in Terminal with gcc, triggering a fair number of errors from `catch.hpp` (that file is massive)
- Checking my gcc version and finding that it's 11.0.0, which I would think would be enough.
- Reading the README (doesn't really offer information on how to run the tests)
- Searching the Discord to see if anybody asked and had answered a relevant question

I don't know if I can write new tests if I can't run the current ones to see how they work.

Reply:
> Likewise, however, if my implementation of a Box-Muller normal is not doing enough, a new comparison would be fair.

With the guards in place the comparison went from 29.7% faster to 16.2% faster.

Reply:
I got the unit tests working with quyykk's help and I added what seem like comparable unit tests in `test_random.cpp` here: 6b248819d9725a4b182b7647be76cc7fa80532aa.

Also, I noticed that the change seems to have broken the checks.  Let me know if a revert commit is needed.